namespaces.config.cue

```diff
@@ -1,9 +1,11 @@
 package holos
 
+import holos "example.com/platform/schemas/holos/v1alpha5"
```

Comment:
This `v1alpha5` package was picked just to disambiguate with the existing package holos.

We may also need to not alias the import to `holos` otherwise we invite conflicts with the `holos` top level field.